meta-client/src/main/java/org/apache/helix/metaclient/factories/MetaClientConfig.java

```diff
@@ -32,10 +34,18 @@ public enum StoreType {
   // Wait for init timeout time until connection is initiated
   private final long _connectionInitTimeoutInMillis;
 
+  // Operation failed because of connection lost will be auto retried if connection has recovered
+  // within timeout time.
+  private final long _operationRetryTimeoutInMillis;
+
```

Comment:
It's a little confusing we have both this one and `MetaClientReconnectPolicy`. Can we combine these two?

Reply:
operationRetryTimeout if for operation retry. Client does not reissue any request until it gets notification that connection is re-established. 
ReconnectPolicy is to define auto reconnect policy when connection to metadata store is lost. 
I think they belongs to different categories.